pkg/segment/results/segresults/segresults.go

```diff
@@ -238,10 +238,6 @@ func (sr *SearchResults) UpdateSegmentStats(sstMap map[string]*structs.SegStats,
 	sr.updateLock.Lock()
 	defer sr.updateLock.Unlock()
 	for idx, measureAgg := range measureOps {
-		if len(sstMap) == 0 {
-			continue
-		}
```

Comment:
Why don't we want to continue here?

Reply:
I forgot to revert this change. Sorry for the oversight. I have addressed this in the latest commit.